### PR TITLE
updating user documentation

### PR DESCRIPTION
**Pull request by anupriyatripathi**
addresses #36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,45 +56,68 @@ wget https://raw.githubusercontent.com/biocore/q2-chemistree/master/q2_chemistre
...
+This gives us a QIIME 2 artifact of type `CSIFolder` that contains probabilities of molecular substructures (total 2936 molecular properties) within in each feature.
+We now generate a contingency table with these probabilities i.e. molecular fingerprints of MS1 features in our experiment. This is of type `FeatureTable[Frequency]`.
 
-Now, we generate the molecular tree using:
+```bash
+qiime chemistree collate-fingerprint --i-csi-result FPs.qza \
+--p-qc-properties \
+--o-collated-fingerprints collated_FPs_QC.qza
+```
+By default, we only use PUBCHEM fingerprints (total 489 molecular properties). Replacing the flag `--p-qc-properties` by `--p-no-qc-properties` retains all the molecular properties in the contingency table. This table is used to generate out hierarchy of molecules!
```
Comment (ElDeveloper) [inline README.md]: Since the default is True (meaning the properties are going to be filtered), I would suggest that you rephrase this to say something along the lines of "Adding `--p-no-qc-properties` retains ...".

File: README.md [hunk 2]
```diff
@@ -50,45 +56,68 @@ wget https://raw.githubusercontent.com/biocore/q2-chemistree/master/q2_chemistre
...
+  --i-collated-fingerprints collated_FPs_QC.qza \
   --p-prob-threshold 0.5 \
   --o-tree demo-chemisTree.qza
 ```
 
-This generates a tree relating the MS1 features in these data based on molecular substructures predicted for MS1 features. This is of type `Phylogeny[Rooted]`.
-
-Finally, note that SIRIUS predicts molecular substructures for a subset of features (typically for 70-90% of all MS1 features) in your experiment (based on factors such as sample type, the quality MS2 spectra, and used-defined tolerances when using the `fingerprint` command). Thus, we need to remove the MS1 features without fingerprints from the feature table with:
+This generates a tree relating the MS1 features in these data based on molecular substructures predicted for MS1 features! This is of type `Phylogeny[Rooted]`. Note that SIRIUS predicts molecular substructures for a subset of features (typically for 70-90% of all MS1 features) in your experiment (based on factors such as sample type, the quality MS2 spectra, and used-defined tolerances such as `ppm-max`, `zodiac-threshold`). Thus, we need to remove the MS1 features without fingerprints from the feature table with:
```
Comment (ElDeveloper) [inline README.md]: I would remove the exclamation mark at the end of the first sentence.

Can you change:
- `ppm-max` -> `--p-ppm-max`.
- `zodiac-threshold` -> `--p-zodiac-threshold`.

File: README.md [hunk 3]
```diff
@@ -50,45 +56,68 @@ wget https://raw.githubusercontent.com/biocore/q2-chemistree/master/q2_chemistre
...
+  --p-ionization-mode positive \
+  --p-java-flags "-Djava.io.tmpdir=/path-to-some-dir/ -Xms16G -Xmx64G" \
+  --o-fragmentation-trees FTs.qza
 ```
 
-First, we generate a contingency table with probabilities of presence of various molecular substructures in an MS1 feature (MS1 fingerprint):
+This generates a QIIME 2 artifact of type `SiriusFolder`. This contains fragmentation trees with candidate molecular formulas for each MS1 features detected in your experiment.
+**Note**: `/path-to-some-dir/` should be a directory where you have write permissions and sufficient storage space. We use -Xms16G and -Xmx64G as the minimum and maximum heap size for Java virtual machine (JVM). If left blank, q2-chemistree will use default JVM flags.
+
+Next, we retain top candidate per MS1 feature using [Zodiac](GitHublink) as follows:
```
[the remark this answers is not in the document]
Comment (anupriyatripathi) [reply README.md]: Thanks! There is no separate URL for zodiac yet, that I'm aware of. Since it ships with Sirius now, I am not mentioning it separately anymore.
